base/src/types.rs

```diff
@@ -347,6 +347,18 @@ pub enum FontScheme {
     None,
 }
 
+// Merge Cell type
+// There will be one MergeCell maintained for every Merge cells that we load
+// It will consist of range_ref : its range reference in String representation ( ex : A1-C2 )
+// merge_col_range : Its tuple having start and end values of columns ( ex: A1-C2 => (1,3))
+// merge row_range : Its tuple having start and end values of rows (ex: A1-C2 => (1,2))
+#[derive(Encode, Decode, Debug, PartialEq, Clone)]
+pub struct MergeCell {
```

Comment:
On IronCalc both rows and columns are `i32`. Now it might not be the best option, and we might one day change the _whole_ code base to a different choice, but for now it is better if we stick to it (and you will not need to do `as u32` all over.

Also, in other places we use `range = [row_start, column_start, row_end, column_end]`. I suggest we use a similar tstructure here.

Reply:
Understood keeping the above struct